frontend/public/components/app.jsx

```diff
@@ -104,12 +104,23 @@ class App extends React.PureComponent {
     this._onNavToggle = this._onNavToggle.bind(this);
     this._onNavSelect = this._onNavSelect.bind(this);
     this._isDesktop = this._isDesktop.bind(this);
+    this._onResize = this._onResize.bind(this);
+    this.previousDesktopState = this._isDesktop();
 
     this.state = {
       isNavOpen: this._isDesktop(),
     };
   }
 
+  componentWillMount() {
+    window.addEventListener('resize', this._onResize);
+  }
+
+  componentWillUnmount() {
+    super.componentWillUnmount();
```

Comment:
I don't think we need the call to `super.componentWillUnmount()` since this isn't extending `SafetyFirst`

Reply:
Updated